package-lock.json

```diff
@@ -9,11 +9,14 @@
       "version": "2.19.0-beta.9",
       "license": "Apache-2.0",
       "dependencies": {
+        "@adobe/aep-rules-engine": "^2.0.2",
+        "@adobe/reactor-cookie": "^1.0.0",
```

Comment:
(non-blocker)
We already depend on `js-cookie`, and `@adobe/reactor-cookie` is just [the thinnest of wrappers on top of `js-cookie`](https://github.com/adobe/reactor-turbine/blob/master/coreModulePackages/cookie/index.js). Just use that.

Reply:
is there an action item you are asking for here @carterworks .  looks like the `@adobe/reactor-query-string` was added back in 2020 by Jon.  And `js-cookie` was added in 2023 by Artur.  I understand both ultimately use `js-cookie`.  Let me know if there is something to do here.